### PR TITLE
Support Service Filtering

### PR DESCRIPTION
**Pull request by aballman**
* Adds "Filter" search box in Project/Services
* FIlters results displayed in Services based on user input

### CODE REVIEW

File: src/components/Project/Secrets/paginator.js [hunk 1]
```diff
@@ -400,6 +401,22 @@ export default class SecretsPaginator extends React.Component {
     this.openDrawer()
   };
 
+  onPanelTableEmpty() {
+    console.log("onPanelTableEmpty!")
```
Comment (amoskyler) [inline src/components/Project/Secrets/paginator.js]: rm console

File: src/components/Project/Secrets/paginator.js [hunk 2]
```diff
@@ -400,6 +401,22 @@ export default class SecretsPaginator extends React.Component {
...
+    return (
+      <Card square={true}>
+        <CardContent>
+          <Typography variant="subheading" style={{ textAlign: "center", fontWeight: 500, fontSize: 23, color: "gray" }}>
+            No Secrets Found!
```
Comment (amoskyler) [inline src/components/Project/Secrets/paginator.js]: This is unrelated to the service view changes, correct?

File: src/components/Utils/Table/index.js [hunk 3]
```diff
@@ -11,9 +11,12 @@ import styles from './style.module.css';
 export default class PanelTable extends React.Component { 
 
   render() {
-    const { title, columns, rows, paginator } = this.props;
+    const { title, columns, rows, paginator, onEmpty } = this.props;
     var self = this;
 
+    console.log(rows.length)
```
Comment (amoskyler) [inline src/components/Utils/Table/index.js]: console stuff

File: src/components/Project/Secrets/paginator.js (hunk 2 above)
Comment (aballman) [reply src/components/Project/Secrets/paginator.js]: No, it's there in case you put a dumb filter in and you have no results. It's a reminder that you have a filter set.
